packages/react-charting/docs/TestPlans/VerticalStackedBarChart/ComponentTests.md

```diff
@@ -0,0 +1,72 @@
+**Vertical Stacked Bar Chart – Component test plan**
+
+**Sub-components: Bar, Line, Legends, Callout, Labels**
+
+1. **Bar: Bar data, Bar color (single/multiple), bar label**
+1. **Line: show/hide line, highlight data points on line and show callout**
+1. **Legends: show/hide legends, highlight the corresponding bar/line on legend hover**
+1. **Callout: Default/custom callout**
+1. **Labels: x-Axis labels default/rotated**
+
+|                                   **Test steps**                                    |                                              **Validation**                                               | **Tool used** |
+| :---------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------: | :-----------: |
+|                             Test 1: [Snapshot testing]                              |                                                                                                           |               |
+|                  - With only data prop, numerical data on x-axis.                   |                                Renders VerticalStackedBar chart correctly                                 |      RTL      |
+|                    - With only data prop, string data on x-axis.                    |                                Renders VerticalStackedBar chart correctly                                 |      RTL      |
```

Comment:
Add a test case for date data on x axis. The chart should not work in this case.